.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - support/*
+  workflow_dispatch:
+  workflow_call:
+
+env:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+  DOTNET_NOLOGO: true
+  DOTNET_GENERATE_ASPNET_CERTIFICATE: false

```

Comment:
super nit: I'd've sorted. Don't bother if you aren't coming back anyhow. Or maybe just don't bother!